packages/expo-dev-launcher/ios/EXRCTAppDelegateInterceptor.mm

```diff
@@ -46,4 +56,13 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge {
   return [self.interceptor sourceURLForBridge:bridge];
 }
 
+- (std::unique_ptr<facebook::react::JSExecutorFactory>)jsExecutorFactoryForBridge:(RCTBridge *)bridge
+{
+#if __has_include(<RNReanimated/REAInitializer.h>) && !RCT_NEW_ARCH_ENABLED
```

Comment:
it seems [reanimated doesn't go through the `reanimated::REAInitializer()` in new arch mode](https://github.com/software-mansion/react-native-reanimated/blob/79d0e5baf4e7aea9bbf27a798472e8c42b6527da/ios/native/RCTAppDelegate%2BReanimated.mm#L1). @tsapeta could you help to double confirm with reanimated team for the behevior?

Reply:
That's true, `REAInitializer` is used and exposed only on iOS Paper. 

Before Reanimated 3.3.0, it was responsible for 3 things:
1. Overwrite an instance of `RCTUIManager` with `REAUIManager`
2. Overwrite an instance of `RCTEventDispatcher` with `REAEventDispatcher`
3. Inject JSI bindings to the RN runtime

In Reanimated 3.3.0, we got rid of 3. as we moved injecting JSI bindings to `installCoreFunctions` (see https://github.com/software-mansion/react-native-reanimated/pull/4523)

Since then, we got rid of 2. as we swizzle `sendEvent` method instead (see https://github.com/software-mansion/react-native-reanimated/pull/4576)

Right now @piaskowyk is working on swizzling methods of `RCTUIManager` so we can get rid of 1. (see `@piaskowyk/UIManager-refactor`).

Once we complete it, `REAInitializer` will be effectively a no-op and we will deprecate it (probably using `#warning`) and eventually it will be completely removed.

Reply:
thanks @tomekzaw for the clear comment. it's great to see your effort on making reanimated better and better 🔥